cs/src/Directory.Build.targets

```diff
@@ -6,9 +6,9 @@
     <Target
         Name="GenerateDocumentation"
         AfterTargets="Build"
-        Condition="'$(GenerateDocs)'=='true' AND '$(TargetFramework)' == 'netcoreapp3.1'"
+        Condition="'$(GenerateDocs)'=='true' AND '$(TargetFramework)' == 'net6.0'"
     >
-        <!-- This target only runs for the .NET 3.1 target; xmldocmd tool currently does not work on .NET 6. -->
+        <!-- This target only runs for the .NET 6 target; xmldocmd does not yet support .NET 8. -->
```

Comment:
`8` or `6`?

Reply:
The comment is correct, this tool doesn't support .NET 8 yet. So it is conditional on `net6.0` target.

I think we don't actually generate these docs anymore though; they were used for the internal docs site.